packages/NotificationCard/src/components/Header/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Header.styles";
+
+function NotificationHeader({ children, ...moreProps }) {
+  return <sc.NotificationHeader {...moreProps}>{children}</sc.NotificationHeader>;
+}
+
+const propTypes = {
+  children: PropTypes.node,
+
+  /** Heading level(1-6) is required. */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]).isRequired,
```

Comment:
why are required? can you not set one as default?

Reply:
Maybe.. but it's required in heading component, so I'm not sure which number should I use for the default

Reply:
Discussed with @mikrotron, it's hard to tell what's the proper heading level for this component in different pages. If we set it as required, it forces the consumer to think about the proper heading level, accessibility will be improved. If someone complains about this, we can revisit for sure!

Reply:
I'm undecided about this.
It could be just an `<h2>` by default (like SidePanel / Modal / Takeover), but that's not always going to be appropriate for this component's context. For those other components it's more likely that `<h2>` is appropriate.
If it's required though, then it forces the consumer to think about the proper heading level (for better accessibility).

Reply:
ok, I don't know what would be worse not setting the header by default or setting to h2 or h3? but I'm not hard on this.